views/layouts/main.php

```diff
@@ -70,7 +76,7 @@
                             <a href="#" class="link">Связаться с нами</a>
                         </li>
                         <li class="menu-item">
-                            <a href="#" class="link">Выход из системы</a>
+                            <a href="/users/logout" class="link">Выход из системы</a>
```

Comment:
Тут не критично, но для ссылок в дальнейшем хорошим правилом является использование хелпера Url. Тогда, например, все ссылки будут корректно работать в том случае, если проект размещен в подпапке (например, если речь идет об админке advanced-приложении, которая доступна по адресу `/admin`):
```suggestion
                            <a href="<?= Url::to(['/users/logout']) ?>" class="link">Выход из системы</a>
```

Reply:
Принял, поправлю.